src/modules/transaction/components/Overview/Overview.js

```diff
@@ -232,7 +232,7 @@ const Overview = () => {
           </div>
         </div>
         <div className={`${styles.column} ${styles.pie}`}>
-          <h2 className={styles.title}>{t('Amount per transaction (LSK)')}</h2>
+          <h2 className={styles.title}>{t('Fee per transaction (LSK)')}</h2>
```

Comment:
I think the designs were updated to fee, but the endpoints are not adapted to fee yet? http://devnet-service.liskdev.net:9901/api/v3/transactions/statistics?limit=7&interval=day

so for now we can keep it `Amount per transaction`?

if the API is updated in future we can change in UI.

Reply:
fixed